File: javascript/programmers/javascript/level1/이상한문자만들기.js
const s = "try hello world";

function solution(s) {
  let answer = "";
  let idx = 0;
  for (let i = 0; i < s.length; i++, idx++) {
    if (s[i] === " ") {
      idx = -1;
      answer += " ";
    }
    if (idx % 2 === 0) {
      answer += s[i].toUpperCase();
    } else {
      answer += s[i].toLowerCase();
    }
  }
  return answer;
}

console.log(solution(s));

File: python/programmers/level1/이상한문자만들기.py
def solution(s):
    arr = s.split(' ')
    idx = 0
    for x in arr:
        answer = ''
        for i in range(len(x)):
            if i % 2 == 0:
                answer += x[i].upper()
            else:
                answer += x[i].lower()
        arr[idx] = answer
        idx += 1

    return ' '.join(map(str, arr))


print(solution("try hello world"))
